server/services/notify/plugindelivery/plugin_delivery.go

```diff
@@ -42,45 +39,17 @@ type PluginAPI interface {
 
 // PluginDelivery provides ability to send notifications to direct message channels via Mattermost plugin API.
 type PluginDelivery struct {
-	botID      string
-	serverRoot string
-	api        PluginAPI
+	botID       string
+	serverRoot  string
+	api         PluginAPI
+	permissions permissions.PermissionsService
 }
 
-func New(botID string, serverRoot string, api PluginAPI) *PluginDelivery {
+func New(botID string, serverRoot string, api PluginAPI, permissions permissions.PermissionsService) *PluginDelivery {
 	return &PluginDelivery{
-		botID:      botID,
-		serverRoot: serverRoot,
-		api:        api,
-	}
-}
-
-func (pd *PluginDelivery) Deliver(mentionUsername string, extract string, evt notify.BlockChangeEvent) error {
```

Comment:
This method had been moved/copied during a previous refactor but left here unintentionally. It is not called.